rb/lib/selenium/webdriver/remote/http/default.rb

```diff
@@ -28,6 +28,27 @@ module Http
         class Default < Common
           attr_accessor :proxy
 
+          attr_accessor :open_timeout
+          attr_accessor :read_timeout
+
+          # Initializes object.
+          # Warning: Setting {#open_timeout} to non-nil values will cause a separate thread to spawn.
+          # Debuggers that freeze the process will not be able to evaluate any operations if that happens.
+          # @param [Numeric] open_timeout - Open timeout to apply to HTTP client.
+          # @param [Numeric] read_timeout - Read timeout (seconds) to apply to HTTP client.
+          def initialize(open_timeout: nil, read_timeout: nil)
+            @open_timeout = open_timeout
+            @read_timeout = read_timeout
+          end
+
+          # Maintaining backward compatibility.
+          # @param [Numeric] value - Timeout in seconds to apply to both open timeout and read timeouts.
+          # @deprecated Please set the specific desired timeout {#read_timeout} or {#open_timeout} directly.
```

Comment:
We should probably not only add `@deprecated` tag but also notify people using this method that it's deprecated, for example using [`Gem::Deprecated`](http://ruby-doc.org/stdlib-2.3.0/libdoc/rubygems/rdoc/Gem/Deprecate.html) or simple [`Kernel#warn`](https://ruby-doc.org/core-2.3.3/Kernel.html#method-i-warn).